docs/other/php.md

```diff
@@ -0,0 +1,16 @@
+Drupal VM defaults to PHP 7.0, but you can also install and use 5.6 or 7.1.
+
+## Ubuntu
+
+Ondřej Surý's PPA for PHP is used to install PHP 7.0, but you can switch to using 5.6 or 7.1 packages by changing `php_version` inside `config.yml` to eg. `"7.1"`.
+
+If you're using Apache with `mod_php` you should also add `libapache2-mod-php7.1` to the `extra_packages` list.
```

Comment:
We need to specify what to do for PHP 5.6 too—or reword this so it is less 7.1-specific.